appinventor/appengine/src/com/google/appinventor/client/actions/BuildComponentAction.java

```diff
@@ -0,0 +1,10 @@
+package com.google.appinventor.client.actions;
```

Comment:
Ditto copyright notice, but maybe we should consider jettisoning this action since the build component functionality was never implemented.